specification/compute/resource-manager/Microsoft.Compute/GalleryRP/stable/2024-03-03/gallery.json

```diff
@@ -0,0 +1,4101 @@
+    },
+    "GalleryInVMAccessControlProfileProperties": {
+      "properties": {
+        "description": {
+          "type": "string",
+          "description": "The description of this gallery inVMAccessControlProfile resources. This property is updatable."
+        },
+        "osType": {
+          "type": "string",
+          "description": "This property allows you to specify the OS type of the VMs/VMSS for which this profile can be used against. Possible values are: 'Windows' or 'Linux'",
+          "enum": [
+            "Windows",
+            "Linux"
+          ],
+          "x-ms-enum": {
+            "name": "OperatingSystemTypes",
+            "modelAsString": false
+          }
+        },
+        "applicableHostEndpoint": {
+          "type": "string",
+          "description": "This property allows you to specify the Endpoint type for which this profile is defining the access control for. Possible values are: 'WireServer' or 'IMDS'",
+          "enum": [
+            "WireServer",
+            "IMDS"
+          ],
+          "x-ms-enum": {
+            "name": "EndpointTypes",
+            "modelAsString": false
+          }
+        }
+      },
+      "required": [
+        "osType",
+        "applicableHostEndpoint"
+      ],
+      "description": "Describes the properties of a gallery inVMAccessControlProfile."
+    },
+    "GalleryInVMAccessControlProfileVersion": {
+      "properties": {
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/GalleryInVMAccessControlProfileVersionProperties"
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "../../../common-types/v1/common.json#/definitions/Resource"
+        }
+      ],
+      "description": "Specifies information about the gallery inVMAccessControlProfile version that you want to create or update."
+    },
+    "GalleryInVMAccessControlProfileVersionUpdate": {
+      "properties": {
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/GalleryInVMAccessControlProfileVersionProperties"
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "#/definitions/UpdateResourceDefinition"
+        }
+      ],
+      "description": "Specifies information about the gallery inVMAccessControlProfile version that you want to update."
+    },
+    "GalleryInVMAccessControlProfileVersionProperties": {
+      "properties": {
+        "mode": {
+          "type": "string",
+          "description": "This property allows you to specify whether the access control rules are in Audit mode, in Enforce mode or Disabled. Possible values are: 'Audit', 'Enforce' or 'Disabled'.",
+          "enum": [
+            "Audit",
+            "Enforce",
+            "Disabled"
+          ],
+          "x-ms-enum": {
+            "name": "AccessControlRulesMode",
+            "modelAsString": false
```

Comment:
This must be set to true, for all enums here.

Reply:
set to true